AppController/lib/datastore_server.rb

```diff
@@ -69,6 +69,15 @@ def self.start(master_ip, db_local_ip, my_ip, table)
   def self.stop(table)
      MonitInterface.stop(:datastore_server)
   end
+  # The following are needed to comply to the djinn calling in
```

Comment:
Why is there a need to distinguish between master and slave at this point? 
Doesn't the method above (self.stop) stop all datastore servers on the machine?